source/Engine.cpp

```diff
@@ -2415,9 +2426,9 @@ void Engine::FillRadar()
 	for(shared_ptr<Ship> &ship : ships)
 		if(ship->GetSystem() == playerSystem)
 		{
-			// Do not show cloaked ships on the radar, except the player's ships.
+			// Do not show cloaked ships on the radar, except the player's ships, and those who should show on radar.
 			bool isYours = ship->IsYours();
-			if(ship->Cloaking() >= 1. && !isYours)
+			if(ship->IsCloaked() && !isYours && !ship->Attributes().Get("cloak radar traceability"))
```

Comment:
~~great, another undocumented feature~~

Reply:
```suggestion
			if(ship->IsCloaked() && !isYours)
```
I thought I removed it